cmd/exporter.go

```diff
@@ -56,6 +57,8 @@ var (
 	enabledEBPFCgroupID          = flag.Bool("enable-cgroup-id", true, "whether enable eBPF to collect cgroup id (must have kernel version >= 4.18 and cGroup v2)")
 	exposeHardwareCounterMetrics = flag.Bool("expose-hardware-counter-metrics", true, "whether expose hardware counter as prometheus metrics")
 	cpuProfile                   = flag.String("cpuprofile", "", "dump cpu profile to a file")
+	memProfile                   = flag.String("memprofile", "", "dump mem profile to a file")
```

Comment:
maybe we can update this settings by default to CI later on so by default we are able to obtain that data?

Reply:
@jichenjc this is a good idea, and we need to think which test we should do that.
Maybe the integration test? Or a new test? This would require not only run the profile but also dump in the logs the information.

Btw, to enable it I was think to use some environment variable to use with `make cluster-sync`. For example:
`ENABLE_CPU_PROFILING=true`

Let's do it in another PR.

Reply:
yes, the environment variable works for me 

and agree we can do this in another PR..